### PR TITLE
Delete all files and update readme with moved notice

### PR DESCRIPTION
**Pull request by panzarino**
Adds an archive warning to the readme and redirects users to the new package location

Repo must be officially archived after merge

### CODE REVIEW

**Review by JessicaSachs — CHANGES_REQUESTED** [review]
The next steps are to also delete all the files in the repository, leaving only this README.md -- can you update the PR to delete all the files?

File: README.md [hunk 1]
```diff
@@ -1,207 +1,9 @@
-# Cypress Webpack Preprocessor [![CircleCI](https://circleci.com/gh/cypress-io/cypress-webpack-preprocessor.svg?style=svg)](https://circleci.com/gh/cypress-io/cypress-webpack-preprocessor) [![semantic-release][semantic-image] ][semantic-url]
+# Cypress Webpack Preprocessor
 
 Cypress preprocessor for bundling JavaScript via webpack
 
-## Installation
+# :warning: THIS REPO HAS BEEN ARCHIVED :warning:
```
Comment (chrisbreiding) [inline README.md:5]:
```suggestion
# :warning: THIS REPO HAS BEEN MOVED :warning:
```

At first glance, "archived" sounds like we're killing off this project, so I think we should say it's been "moved".

**Review by chrisbreiding — DISMISSED** [review]

**Comment by panzarino** [comment]
Is there a better emoji or formatting that we could use here? I'm afraid that this comes off as if we deprecated the project: 

Screen Shot 2020-09-29 at 2 47 26 PM

**Comment by JessicaSachs** [comment]
Yeah, a moving truck

**Review by JessicaSachs — APPROVED** [review]

**Review by brian-mann — APPROVED** [review]

**Review by chrisbreiding — APPROVED** [review]

**Pull request merged**